### PR TITLE
Fixes wizard refund glitch

### PR DESCRIPTION
**Pull request by datlo**
Laughter demon currently refunds 2 spell points, when it costs 1 point, this fixes that.

🆑 
Fix: Fixed an exploit letting wizards farm spellpoints.
\ 🆑

### CODE REVIEW

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -534,8 +534,10 @@
 
 	if(istype(O, /obj/item/antag_spawner/slaughter_demon))
 		to_chat(user, "<span class='notice'>On second thought, maybe summoning a demon is a bad idea. You refund your points.</span>")
-
-		uses+=2
+		if(istype(O, /obj/item/antag_spawner/slaughter_demon/laughter))
+			uses+=1
```
Comment (Fox-McCloud) [inline code/game/gamemodes/wizard/spellbook.dm]: spacing.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 2]
```diff
@@ -534,8 +534,10 @@
 
 	if(istype(O, /obj/item/antag_spawner/slaughter_demon))
 		to_chat(user, "<span class='notice'>On second thought, maybe summoning a demon is a bad idea. You refund your points.</span>")
-
-		uses+=2
+		if(istype(O, /obj/item/antag_spawner/slaughter_demon/laughter))
+			uses+=1
+		else
+			uses+=2
```
Comment (Fox-McCloud) [inline code/game/gamemodes/wizard/spellbook.dm]: 👀

Comment (SamHPurp) [reply code/game/gamemodes/wizard/spellbook.dm]: ❄️

**Comment by variableundefined** [comment]
In the future remember to add spacing between a variable and an operator for clarity sake e.g. "uses += 2" instead of "uses+=2"

**Comment by datlo** [comment]
will do thanks!

**Pull request merged**